scipy/stats/kde.py

```diff
@@ -492,15 +494,26 @@ def _compute_covariance(self):
         covariance_factor().
         """
         self.factor = self.covariance_factor()
-        # Cache covariance and inverse covariance of the data
-        if not hasattr(self, '_data_inv_cov'):
+        # Cache covariance and cholesky decomp of covariance of the data
+        if not hasattr(self, '_data_cov_chol'):
             self._data_covariance = atleast_2d(np.cov(self.dataset, rowvar=1,
                                                bias=False))
-            self._data_inv_cov = linalg.inv(self._data_covariance)
+
+            # cho_solve returns a pair (ndarray, bool)
+            self._data_cov_chol = linalg.cho_factor(self._data_covariance)
 
         self.covariance = self._data_covariance * self.factor**2
-        self.inv_cov = self._data_inv_cov / self.factor**2
-        self._norm_factor = sqrt(linalg.det(2*pi*self.covariance)) * self.n
+
+        # This is very bad and should be deprecated.
+        self.inv_cov = linalg.inv(self._data_covariance) / self.factor**2
```

Comment:
I'd also like this attribute to be removed, but I can't do that without breaking backwards compatibility.  Is there a way to deprecate an attribute?